src/core_atmosphere/physics/physics_mmm/bl_ysu.F

```diff
@@ -892,6 +893,7 @@ subroutine bl_ysu_run(ux,vx,tx,qvx,qcx,qix,nmix,qmix,p2d,p2di,pi2d,     &
        endif
      enddo
    enddo
+
```

Comment:
Would it be preferable to remove this blank line that was added?

Reply:
Done. I must have added this blank line when debugging the sourcecode and adding print statements. This extra line has been removed as suggested.